packages/captp/src/captp.js

```diff
@@ -431,6 +423,7 @@ export const makeCapTP = (
     async CTP_DROP(obj) {
       const { slotID } = obj;
       slotToVal.delete(slotID);
+      send({ type: 'CTP_DROP', slotID: reverseSlot(slotID), epoch });
```

Comment:
Adding this just ping-pongs `CTP_DROP` calls back and forth between endpoints.  That's not the desired behaviour.

Reply:
good to know. thanks.